Dockerfile

```diff
@@ -5,9 +5,10 @@ RUN apt-get install -y --no-install-recommends npm
 COPY . /app
 
 RUN npm install --production
-RUN ./bin/peep.py install --no-cache-dir -r requirements/dev.txt
-RUN ./bin/peep.py install --no-cache-dir -r requirements/prod.txt
-RUN ./bin/peep.py install --no-cache-dir -r requirements/docker.txt
+RUN ./bin/pipstrap.py
```

Comment:
That's only true after this is merged, so it's not been run there yet. We can remove this later though.